src/plugins/cosmos/components/modals/Staking/views/Stake.tsx

```diff
@@ -157,12 +140,12 @@ export const Stake = ({
           py='8px'
           mb={6}
           assetSymbol={asset.symbol}
```

Comment:
In case you want to do this inside this PR, you should fill the `asset.icon` as an `assetIcon` props here and add the prop to `AssetHoldingsCard`

For reference: https://github.com/shapeshift/web/pull/1388

Reply:
Let's keep it separate since it's not related to wiring up unchained data and calls